### ISSUE TITLE
General: Ensure Style Guide is keyboard accessible from navigation to main content

### ISSUE THREAD

**Issue by davezen1** [US-CBP/cbp-style-guide#33]
Per our a11y expert: 
Items in the style guide are not keyboard accessible.  I can’t get focus to move out of the navigation pane and into the actual style guide.

**Comment by davezen1** [comment]
@cmterrylynn Let's revisit after we carry over the a11y fixes from the cbp theme kitchen sink.  In general pages that have actionable items like buttons, links, etc. should be accessible. And pages like Buttons are keyboard accessible but Accordions are not as yet.

**Comment by yellowzapp** [comment]
going to close this for now. Will revisit after DHS OAST gives new guidance on Trusted Tester compliance in October 2018.
